Applications/Hermes/hermesflaskserver.py

```diff
@@ -0,0 +1,177 @@
+import os
+import hermesobject as hermesobject
+import ConfigParser
+import json
+import base64
+import cPickle
+
+from flask import Flask
+from flask import request
+
+# Open configuration INI
+config = ConfigParser.RawConfigParser()
+filename = os.path.abspath("hermes.ini")
+config.read(filename)
+
+# Global variables
+dictmsgobj = {}
+
+
+def configparse():
+    """ Parses configuration file for callsigns and node ID and returns them in JSON format"""
+    local = {}
+    num = int(config.get('PROXY', 'UNITS'))
+    units = range(0, num)
```

Comment:
range(num), you could also just forgo using num at all and directly use the config call.